### PR TITLE
P0 - Avatar & Badge components

### PR DESCRIPTION
**Pull request by HarshithaKonduru214**
Please review the below 2 components. I have attached the output screenshots below.
image

image

### CODE REVIEW

File: style_badge.css [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    margin-top: 3rem;
+  }
+  
+  .text {
+    font-size: 2rem;
+  }
+
+  .button {
+      font-size: 1.5rem;
+      background-color: #7088D2;
```
Comment (NtshVrm) [inline style_badge.css:25]: use a variable instead of the hex code

Comment (NtshVrm) [inline style_badge.css]: use a variable instead of the hex code

**Review by NtshVrm — APPROVED** [review]
Looks good👍

**Review by ayush65 — APPROVED** [review]
Looks Perfect

**Pull request merged**